src/Tx_main.cpp

```diff
@@ -150,6 +153,39 @@ void SendVersionResponse()
   msp.sendPacket(&out, &Serial);
 }
 
+void HandleConfigMsg(mspPacket_t *packet)
+{
+  uint8_t key = packet->readByte();
+  uint8_t value = packet->readByte();
+  switch (key)
+  {
+    case MSP_ELRS_BACKPACK_CONFIG_TLM_MODE:
+      switch (value)
+      {
+        case 0: // TELEM_MODE_OFF
+          config.SetTelemMode(TELEM_MODE_OFF);
+          config.SetWiFiService(WIFI_SERVICE_UPDATE);
+          config.SetStartWiFiOnBoot(false);
+          config.Commit();
+          break;
+        case 1: // TELEM_MODE_RECEIVER
```

Comment:
Why is this one called RECEIVER if it's doing ESPNow?

Reply:
From a user point of view, the backpack receiver is usually a "receiver". Gets flashed that as a "tracker backpack" or similar. Yes, the transport is ESPNOW, but it's still going to be considered as a "receiver" by users.

Reply:
I disagree.

From a developer POV, we understand what "ESPNOW" is, so yes, its clearer for us.
From a user POV though, most ppl don't know or care what ESPNOW is, they just want their telem to go to their "receiver thingy".

For the sake of getting through code review though, Ive changed it to what you've asked (on both PRs), even though I feel like this is not a positive suggestion.